nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/eventhub/ConsumeAzureEventHub.java

```diff
@@ -76,6 +76,7 @@
 import java.util.concurrent.TimeUnit;
```

Comment:
com.microsoft.azure.eventhubs.ConnectionStringBuilder is an unused import above.

Reply:
removed.